workspace/CardFantasyCore/src/cfvbaibai/cardfantasy/game/launcher/GameLauncher.java

```diff
@@ -201,13 +201,7 @@ public static BossGameResult playBossGame(GameSetup setup) {
             result.setLastDetail(trialResult);
             return result;
         }
-        int gameCount = 10000 / trialResult.getRound();
-        if (gameCount < 100) {
-            gameCount = 100;
-        }
-        if (gameCount > 1000) {
-            gameCount = 1000;
-        }
+        int gameCount = setup.getGameCount();
 
```

Comment:
The original algorithm is to avoid long boss battle from taking too much system resources by doing a trial game and calculate the game count out of the time the trial game takes. Please do not change it back to fixed game count algorithm.